packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -43,7 +42,24 @@ const EDiscovery = WebexPlugin.extend({
       service: 'ediscovery',
       resource: 'reports',
       body
-    });
+    })
+      .catch((reason) => {
+        // This value correspondes to the ediscovery error code INVALID_USER_EMAILS_LIST_IN_MESSAGE
+        if (reason.body.errorCode === 14400001) {
+          try {
+            const invalidEmails = JSON.parse(reason.body.message);
+
+            if (Array.isArray(invalidEmails) && invalidEmails.length) {
+              return Promise.reject(new InvalidEmailAddressError(invalidEmails));
+            }
```

Comment:
Should you handle the scenario where it's not an array or the array is empty?

Likely you'll want to log a warn/error.